aptos-move/framework/cached-packages/src/aptos_stdlib.rs

```diff
@@ -3,29 +3,15 @@
 
 #![allow(unused_imports)]
```

Comment:
why is not this called aptos_stdlib_builder?

Reply:
hahaha, probably should rename these to just lib and no builder.